tests/test/test.js

```diff
@@ -92,12 +92,72 @@ assert.equalEpsilon = function(actual, expected) {
     if (typeof actual !== 'number') {
         throw new Error("Expected " + actual + " to be a number.");
     }
-    let epsilon = Math.abs(expected * 1e-6);
+    let epsilon = 1e-6;
+    epsilon = Math.max(epsilon, Math.abs(expected * epsilon));
     if (Math.abs(actual - expected) > epsilon) {
         throw new Error("Expected " + actual + " to equal " + expected);
     }
 };
 
+assert.equalEpsilonArray = function(actual, expected) {
+    const length = expected.length;
+    assert.strictEqual(actual.length, length);
+
+    for (let i = 0; i < length; ++i) {
+        assert.equalEpsilon(actual[i], expected[i]);
+    }
+};
+
+function getAccessorData(gltfPath, asset, accessorIndex, bufferCache) {
+    // This is only for testing exporter output, it does not handle enough
+    // cases to parse arbitrary glTF files from outside this test suite.
+
+    const accessor = asset.accessors[accessorIndex];
+    const bufferView = asset.bufferViews[accessor.bufferView];
+    const bufferIndex = bufferView.buffer;
+    let bufferData;
+    if (bufferCache[bufferIndex] !== undefined) {
+        bufferData = bufferCache[bufferIndex];
+    } else {
+        const buffer = asset.buffers[bufferIndex];
+        const binPath = path.resolve(gltfPath, '../' + buffer.uri);
+        bufferData = fs.readFileSync(binPath);
+        bufferCache[bufferIndex] = bufferData;
+    }
+
+    const bufferViewByteOffset = bufferView.byteOffset || 0;
+    const bufferViewData = bufferData.slice(bufferViewByteOffset, bufferViewByteOffset + bufferView.byteLength);
+
+    let componentSize;
+    switch (accessor.componentType) {
+    case 5126:  // FLOAT
+        componentSize = 4;
+        break;
+    default:
+        throw new Error("Untested accessor componentType " + accessor.componentType);
+    }
+
+    let numElements;
+    switch (accessor.type) {
+    case 'SCALAR':
+        numElements = 1;
+        break;
+    default:
+        throw new Error("Untested accessor type " + accessor.type);
+    }
+
+    // Note that accessor.stride is not handled yet.
+    let count = accessor.count;
+    let stride = componentSize * numElements;
+    let byteOffset = accessor.byteOffset || 0;
+
+    let accessorData = [];
```

Comment:
nit: all of the variables above could be `const`, any reason to use `let`?

Reply:
Fixed.